### PR TITLE
:sparkles: Introducing ActionsColumn Component in Assessment Table

### PR DESCRIPTION
**Pull request by ibolton336**
- This PR introduces the ActionsColumn component in the assessment table 
Screenshot 2023-08-24 at 9 48 38 PM

### CODE REVIEW

File: client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx [hunk 1]
```diff
@@ -632,153 +634,112 @@ export const ApplicationsTable: React.FC = () => {
...
+                      {application.tags ? application.tags.length : 0}
+                    </Td>
+                    <Td isActionCell>
+                      <Button
+                        variant="plain"
+                        icon={<PencilAltIcon />}
+                        onClick={() =>
+                          setSaveApplicationModalState(application)
+                        }
+                      ></Button>
```
Comment (mturley) [inline client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx:713]: You can make this empty Button tag self-closing

File: client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx [hunk 2]
```diff
@@ -632,153 +634,112 @@ export const ApplicationsTable: React.FC = () => {
...
+                            onClick: () =>
+                              setApplicationsToDelete([application]),
+                          },
+                          {
+                            title: t("actions.manageDependencies"),
+                            onClick: () =>
+                              setApplicationDependenciesToManage(application),
+                          },
+                        ]}
+                      ></ActionsColumn>
```
Comment (mturley) [inline client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx:737]: Same here with ActionsColumn

**Review by mturley — APPROVED** [review]
Awesome! Nitpicks with self-closing tags, take them or leave them.

TIL about the ActionsColumn component, I suppose we should update the other composable tables to use it too

**Comment by ibolton336** [comment]
> Awesome! Nitpicks with self-closing tags, take them or leave them.
> 
> TIL about the ActionsColumn component, I suppose we should update the other composable tables to use it too

+1 Opened this issue to track https://github.com/konveyor/tackle2-ui/issues/1318

**Pull request merged**